### ISSUE TITLE
Linuxbrew install of 5.2.0 fails

### ISSUE THREAD

**Issue by mattsches** [dundee/gdu#66]
Not sure if this is the right place to report this issue, but when I try to install or upgrade to 5.2.0 I get:

```bash
brew install -f gdu
==> Downloading https://ghcr.io/v2/linuxbrew/core/gdu/manifests/5.2.0
curl: (22) The requested URL returned error: 404                              

Error: Failed to download resource "gdu_bottle_manifest"
Download failed: https://ghcr.io/v2/linuxbrew/core/gdu/manifests/5.2.0
```

Up until version 5.1.1 I didn't have any problems installing.

**Comment by bergtholdt** [comment]
Same here :(

**Comment by dundee** [comment]
Looks like some problem in linuxbrew. I have created issue there - https://github.com/Homebrew/linuxbrew-core/issues/23920

**Comment by dundee** [comment]
It's fixed with version 5.3.0.
